src/api/app/models/label_template.rb

```diff
@@ -27,6 +27,11 @@ class LabelTemplate < ApplicationRecord
 
   #### Instance methods (public and then protected/private)
 
+  # Random color from 0x000000 - 0xffffff
+  def set_random_color
+    self.color = "##{(rand * 0xffffff).to_i.to_s(16)}"
```

Comment:
`.rjust(6, '0')}` was kind of crucial here, otherwise it doesn't pad out hex smaller than `0x100000` with 0s at the start

Reply:
Ooh right, I fixed that, but I managed to remove it from the code... Good catch!